test/test_pooling.py

```diff
@@ -486,10 +486,6 @@ def f():
         self.assertEqual(nthreads, self.n_passed)
         self.assertTrue(len(cx_pool.sockets) > 1)
 
-    def test_max_pool_size_zero(self):
-        with self.assertRaises(ValueError):
-            rs_or_single_client(maxPoolSize=0)
```

Comment:
We should reinstate this test and assert that `maxPoolSize=0` works as it's expected to.

Reply:
Done.